src/drivers/src/stm32f4xx.cpp

```diff
@@ -38,21 +41,21 @@ HAL_StatusTypeDef CANlib_TransmitFrame(Frame *frame, CANlib_Bus_T bus) {
   CAN_TxHeaderTypeDef pHeader;
   uint32_t pTxMailbox = 0;
 
-  pHeader.DLC                = frame->dlc;
-  pHeader.StdId              = frame->id;
-  pHeader.IDE                = frame->extended ? CAN_ID_EXT : CAN_ID_STD;
+  pHeader.DLC                = frame.dlc;
```

Comment:
You seem to have all the driver files duplicated in `src/drivers` and `src/src/drivers`